### ISSUE TITLE
AgentQL

### ISSUE THREAD

**Issue by nearestnabors** [michaelrambeau/bestofjs#1028]
**Please briefly describe the project you'd like to add to _Best of JS_?**
TLDR: AgentQL is a query language and parser that fetches data and interactive elements from web pages for scraping, automation, and agentic implementations.

AgentQL is a suite of extraction tools for parsing HTML. AgentQL is an [AI-powered](https://docs.agentql.com/intro/under-the-hood) query language and parser that automates interactions with browsers and extracts data from web pages with precision and at scale. AgentQL is able to find what you’re looking for on any page using natural language and AI, regardless of how the page’s structure might change over time. You can think of it as a “smart” alternative to CSS selectors or XPath—capable of pinpointing the exact data and elements you need from even the most complex DOM tree. Getting started with the JS SDK [takes five minutes](https://docs.agentql.com/quick-start?lang=js&type=async), or you can try out our convenient [playground](https://playground.agentql.com/).

**What is the GitHub repository of the project you'd like to add to _Best of JS_?**
https://github.com/tinyfish-io/agentql/

**Before submitting the request to add a new project, please check the following points:**

- [x] The project is related to the web platform or Node.js
- [x] The source code is on GitHub and project has more than 100 stars on GitHub
- [x] The project has thorough documentation in English
- [x] The project is not deprecated and actively maintained

**How would this project be tagged? (Pick one or several existing tags from this list: https://bestofjs.org/tags)**
[HTML & text parsing](https://bestofjs.org/projects?page=1&limit=30&sort=total)
[Scraping](https://bestofjs.org/projects?page=1&limit=30&tags=scraping&sort=total)
[Browser automation](https://bestofjs.org/projects?page=1&limit=30&tags=scraping&sort=total)
[AI](https://bestofjs.org/projects?page=1&limit=30&tags=ai&sort=total)

**Also you can suggest new tags to add if you think _Best of JS_ is missing useful tags.**

**Comment by michaelrambeau** [comment]
Thank you for the recommendation @rachelnabors AgentQL is available on _Best of JS_, please check:

https://bestofjs.org/projects/agentql
